sdk/eventhub/Microsoft.Azure.EventHubs/tests/Client/DataBatchTests.cs

```diff
@@ -75,30 +75,33 @@ public async Task SendWithPartitionKey()
         [DisplayTestMethodName]
         public async Task SendingPartitionKeyBatchOnPartitionSenderShouldFail()
         {
-            var ehClient = EventHubClient.CreateFromConnectionString(TestUtility.EventHubsConnectionString);
-            var partitionSender = ehClient.CreatePartitionSender("0");
-            try
+            await using (var scope = await EventHubScope.CreateAsync(4))
```

Comment:
We don't need 4 partitions for this test.   Looks like 1 would be ideal.    Can we please review the partition counts that we're using to ensure that we're only requesting the number of partitions that a test uses, or that are needed to validate the scenario?

Reply:
Leaving one comment here, but I'd appreciate if we could do that diligence across all of the tests.

Reply:
For some tests would access partition 1 or 2, we create 2 or 3 partitions. Exception this case, all other tests updated to only use one 1 partition.